website/docs/best-practices.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Best Practices
+---
+
+There are some patterns and practices you may want to consider when using Copa to patch images. Remember that these are suggestions that may not fit into your workflow, but we think that staying as close as possible to these practices offers the best experience with Copa.
+
+## Patch from the Base Image
+When patching vulnerabilities in an image, it helps to always work from the base image. For example, say you have an image tagged `nginx:1.24.0` that contains a vulnerability. You run Copa to patch the image and produce a new image tagged `nginx:1.24.0-1`. Then if another vulnerability shows up in your `nginx:1.24.0-1` image, you should again patch from the base `nginx:1.24.0` image. This will help prevent the buildup of patch layers.
+
+## Tagging
+There are a couple possible patterns that you could follow when tagging patched images.
+
+### Incremental
+The first approach you could take is incrementing a number you append to the end of an image tag. For example, if you have an image tagged `nginx:1.24.0`, following patches would be tagged as `nginx:1.24.0-1`, `nginx:1.24.0-2`, `nginx:1.24.0-3`, and so on.
+
+With this pattern you are always explicitly aware of the patch state of the image you are using. The downside is that dependabot is currently unable to act on these new incremental tags.
```

Comment:
> dependabot is currently unable to act on these new incremental tags.

should we say something like "bump to patched images from unmodified images or bump from one patched to next" to be more explicit?